examples/AspNetCore/appsettings.json

```diff
@@ -28,5 +28,15 @@
   },
   "AspNetCoreInstrumentation": {
     "RecordException": "true"
+  },
+  "Kestrel": {
```

Comment:
Is this needed? IIRC these are the default values.

Reply:
i had trouble getting it to use a fixed one (IIRC, it was doing random ports each time).
I'll check this, and remove if not needed.